.changeset/pretty-otters-whisper.md

```diff
@@ -0,0 +1,69 @@
+---
+'@backstage/plugin-techdocs-node': minor
+---
+
+Add azurite support in techdocs through `techdocs.publisher.azureBlobStorage.azuriteConnectionString`
+
+These changes are **required** to `plugins/techdocs-node/src/stages/publish/azureBlobStorage.ts` and `docs/features/techdocs/configuration.md`
```

Comment:
Everything below, including this line, should be deleted from the changeset.

The purpose of a changeset files is only to generate a changelog entry for a given package _when making a release_ and ensuing that the required semver version bump happens. Its text body should inform the end user who's consuming the plugin what was changed. There's no further action on their part; the code below just replicates what the pull request contains and they don't have any `plugins/techdocs-node` folder